docker/common-services.yaml

```diff
@@ -0,0 +1,128 @@
+services:
+  teams-api-common:
+    image: voxel51/fiftyone-teams-api:v1.4.1
+    environment:
+      AUTH0_CLIENT_ID: ${AUTH0_CLIENT_ID}
+      AUTH0_AUDIENCE: ${AUTH0_AUDIENCE}
+      AUTH0_DOMAIN: ${AUTH0_DOMAIN}
+      AUTH0_API_CLIENT_ID: ${AUTH0_API_CLIENT_ID}
+      AUTH0_API_CLIENT_SECRET: ${AUTH0_API_CLIENT_SECRET}
+      FIFTYONE_DATABASE_NAME: ${FIFTYONE_DATABASE_NAME}
+      FIFTYONE_DATABASE_URI: ${FIFTYONE_DATABASE_URI}
+      FIFTYONE_ENCRYPTION_KEY: ${FIFTYONE_ENCRYPTION_KEY}
+      FIFTYONE_ENV: ${FIFTYONE_ENV}
+      FIFTYONE_INTERNAL_SERVICE: true
+      GRAPHQL_DEFAULT_LIMIT: ${GRAPHQL_DEFAULT_LIMIT}
+      LOGGING_LEVEL: ${API_LOGGING_LEVEL:-INFO}
+      MONGO_DEFAULT_DB: ${FIFTYONE_DATABASE_NAME}
+      # If you are routing through a proxy server you will want to set
+      #  HTTP_PROXY_URL, HTTPS_PROXY_URL, and NO_PROXY_LIST in your .env
+      #  then add the following environment variables to your
+      #  `compose.override.yaml`
```

Comment:
As a side note - looks like I recommend the use of `compose.override.yaml` here too...